src/WebUI/Utils.ts

```diff
@@ -100,9 +100,7 @@ export class Utils {
             && podTemplate.spec
             && podTemplate.spec.containers
             && podTemplate.spec.containers.length > 0) {
-            const containersCount = podTemplate.spec.containers.length;
-            const defaultImage = podTemplate.spec.containers[0].image;
-            const imageText: string = containersCount > 1 ? localeFormat(Resources.MoreImagesText, defaultImage, containersCount - 1) : defaultImage;
+            const imageText: string = Utils.getImageText(podTemplate.spec.containers);

```

Comment:
we can remove following

&& podTemplate.spec.containers
            && podTemplate.spec.containers.length > 0

and let getImageText() do all the checks